source/opt/passes.h

```diff
@@ -74,6 +75,15 @@ class EliminateDeadConstantPass : public Pass {
   bool Process(ir::Module*) override;
 };
 
+// The optimization pass to deduplicate the constants. Front-end constants with
+// same value will be unified and only one constants will be kept for each
```

Comment:
`... deduplicate constants. Normal constants of the same value (with identical bit pattern) ...`

Also should mention the effects of `Decoration`s.